client/src/screens/SignIn/SignIn.jsx

```diff
@@ -1,54 +1,42 @@
 import { useEffect, useState } from 'react';
 import { useNavigate } from 'react-router-dom';
 // assets
-import { useDispatch } from 'react-redux';
+import { useDispatch, useSelector } from 'react-redux';
 import { loginUser } from '../../services/redux/slices/uiActions.js';
 import '../SignUp/SignUp.scss';
-import store from '../../services/redux/store.js'
-import { GenericModal } from '../../components/Modal/GenericModal';
 import { handleChange } from '../../services/utils/formHandlers';
 import { SingleActionButton } from '../../components/Button/SingleActionButton';
-import { checkEmailAuth, signOut } from '../../services/api/users';
+import { checkEmailAuth, signOut, verify } from '../../services/api/users';
 
 export const SignIn = () => {
   const navigate = useNavigate();
   const dispatch = useDispatch();
-  const [noAccountError, setNoAccountError] = useState(false);
-  const [showModal, setShowModal] = useState(false);
+  const { _id } = useSelector((state) => state.ui.user);
```

Comment:
i can edit this bit myself, but in another case when we are destructuring something as vague as `_id`, during the destructure we can rename it to a more semantic variable. 
``` const { _id:userId } = useSelector(state => state.ui.user) ```
will rename the destructured variable to whatever is on the right of the colons.